feature-libs/product-configurator/rulebased/core/facade/configurator-groups.service.ts

```diff
@@ -269,6 +269,28 @@ export class ConfiguratorGroupsService {
     return this.getNeighboringGroupId(owner, 1);
   }
 
+  /**
+   * Returns the description of the next group.
+   *
+   * @param {Configurator.Configuration} configuration - Configuration
+   * @returns {string} the description of the next group
+   */
+  getNextGroupDescription(configuration: Configurator.Configuration): string {
+    let description: string = '';
+    this.getNextGroupId(configuration.owner)
+      .pipe(take(1))
```

Comment:
Let's extract this into a common (protected) function and use it also in getPreviousGroupDescription...

Reply:
done